client/src/components/screens/PatientVariant/components/SearchInput.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import intl from 'react-intl-universal';
+import {
+  AutoComplete, Col, Input, Row, Tag, Typography,
+} from 'antd';
+import IconKit from 'react-icons-kit';
+import {
+  ic_search,
+  ic_done,
+} from 'react-icons-kit/md';
+
+function getHighlightSearch(value: string, searchValue: string) {
+  const regex = new RegExp(searchValue, 'i');
+  let tempoValue = value;
+  const highlightValue: string[] = [];
+  const highlightPart = value.split(regex);
+  let haveMoreValue = true;
+
+  while (haveMoreValue) {
+    const matchValue = tempoValue.match(regex);
+    if (matchValue) {
+      highlightValue.push(matchValue[0]);
+      // @ts-ignore
+      tempoValue = tempoValue.slice(matchValue.index + matchValue[0].length);
+    } else {
+      haveMoreValue = false;
+    }
+  }
+
+  return highlightPart.map((stringPart, index) => (
+    <>
+      { index === 0 ? null : <span className="highlight">{ highlightValue[index - 1] }</span> }{ stringPart }
```

Comment:
No scss file has been included, but the `highlight` className is used?
Create a new scss file, and copy all the definitions you use from `navigation.scss`